rllib/utils/framework.py

```diff
@@ -64,7 +64,7 @@ def try_import_tf(error: bool = False):
     # that uses them: del os.environ["RLLIB_TEST_NO_TF_IMPORT"]
     if "RLLIB_TEST_NO_TF_IMPORT" in os.environ:
         logger.warning("Not importing TensorFlow for test purposes")
-        return None, None, None
+        return None, _TFStub(), None
```

Comment:
Since we already have an instance:
`return None, tf_stub, None`

Reply:
done